plugins/goodix-tp/fu-goodixtp.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2023 Goodix.inc <xulinkun@goodix.com>
+// SPDX-License-Identifier: LGPL-2.1+
+
+enum GoodixtpIcType {
+    None,
+    Phoenix,
+    TypeNanjing,
+    Mousepad,
+    Normandyl,
+    Berlinb,
+    Yellowstone,
+}
+
+#[derive(Parse)]
+struct GoodixBrlbHdr {
+    firmware_size: u32le,
+    checksum: u32le,
+    _unknown: [u8; 19],
+    vid: u16be,
```

Comment:
@xulinkun is this definitely big endian? I thought all of brlb was LE?

Reply:
En... except vid is BE, the others is LE in brlb.